profiles/common/themes/ec_resp/template.php

```diff
@@ -1137,6 +1142,15 @@ function ec_resp_menu_local_tasks(&$variables) {
  */
 function ec_resp_form_alter(&$form, &$form_state, $form_id) {
   switch ($form_id) {
+    case 'nexteuropa_europa_search_search_form':
+      if (theme_get_setting('enable_interinstitutional_theme')) {
+        $form['search_input_group']['europa_search_submit']['#type'] = 'image_button';
+        $form['search_input_group']['europa_search_submit']['#src'] = drupal_get_path('theme', 'ec_resp') . '/images/search-button.gif';
+        $form['search_input_group']['europa_search_submit']['#attributes']['class'][] = 'btn btn-default btn-small';
```

Comment:
This doesn't seem correct. If the `[#attributes][class]` are an array, then each class should be a separate item in the array. Here a whole bunch of classes are added in a single space-separated string.

So you should probably use `array_merge()` here:
```
$form[...]['class'] = array_merge($form[...]['class'], ['btn', 'btn-default', '...']);
```